indico/modules/rb/client/js/common/rooms/RoomDetailsModal.jsx

```diff
@@ -171,6 +192,23 @@ function RoomDetails({bookRoom, room, availability, attributes}) {
               trigger={<Icon name="info circle" className="legend-info-icon" />}
               content={<TimelineLegend labels={legendLabels} compact />}
             />
+            <Button
+              as="a"
+              href={bookingsPath}
+              basic
+              size="tiny"
+              compact
+              color="blue"
+              styleName="all-bookings"
+              onClick={evt => {
+                if (evt.button !== 1) {
```

Comment:
shouldn't this be `=== 0`? If we somehow end up getting in here for a right click (button 2) we don't want to do anything...